source/extensions/filters/network/common/redis/codec_impl.cc

```diff
@@ -316,8 +316,8 @@ RespValue::CompositeArray::CompositeArrayConstIterator::operator++() {
   return *this;
 }
 
-bool RespValue::CompositeArray::CompositeArrayConstIterator::operator!=(
-    const CompositeArrayConstIterator& rhs) const {
+bool RespValue::CompositeArray::CompositeArrayConstIterator::
+operator!=(const CompositeArrayConstIterator& rhs) const {
```

Comment:
I think this change is not necessary? This is causing the format error.

Reply:
Sorry for delay, was on PTO.
This change was done by one of the linters as without it other test was failing.
Let me try to run all linters again.

Reply:
/wait